api/src/main/java/com/epam/pipeline/controller/search/SearchController.java

```diff
@@ -66,4 +72,19 @@ public Result<SearchResult> search(@RequestBody ElasticSearchRequest searchReque
     public Result<FacetedSearchResult> facetedSearch(@RequestBody final FacetedSearchRequest searchRequest) {
         return Result.success(searchManager.facetedSearch(searchRequest));
     }
+
+    @PostMapping(value = "/search/export")
```

Comment:
since we export faceted search result, let's use `/search/facet/export` for API method